Test-Rack-Software/config.py

```diff
@@ -2,32 +2,41 @@
 # defined avionics interface. This means that by defining which stack you want to use in the
 # next couple of lines, this file automatically tells `main.py` which file
 # to access.
+import util.dynamic_url
 from util.communication.communication_interface import CommunicationChannelType
+import util.os_interface
 
 # Metadata needs to be imported first, so the avionics system knows what data to use for startup.
 # Import all metaconfigs
 import tars_rack.platform_meta as TARSmkIVmeta  # noqa
-
+import midas_rack.platform_meta as MIDASmkImeta  # noqa
+import dummy_rack.platform_meta as DUMMYmeta  # noqa
 
 # == EDIT THE VARIABLE BELOW TO CHANGE WHICH METADATA IS USED ==
-use_meta = TARSmkIVmeta.meta
+use_meta = MIDASmkImeta.meta

```

Comment:
Is there a way to make it so that Midas meta and interface are combined into one variable so that we don't need to change it in 2 places?

Reply:
Not that I can think of, it is a bit weird but combining them into one and using regular imports causes circular dependency errors. Maybe we can play around with having the interface filename in the `meta` file and dynamically `__import__`'ing it.

Reply:
I'll look into it